<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cost_modified</class>
 <widget class="QDialog" name="cost_modified">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>695</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="year_widget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>36</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>17</y>
     <width>891</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>100</y>
     <width>901</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hun_Y">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>40</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>년도 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>64</y>
     <width>891</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>617</y>
     <width>901</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>3</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cost_cancel_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>74</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="hap_total">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>불러온 지출 항목 총액 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cost_hap_widget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>78</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hun_month">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>37</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>월 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_widget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hun_week">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>36</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>몇번째 주 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="week_widget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>구분</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_combo_widget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>36</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="cost_modi_tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>891</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <column>
    <property name="text">
     <string>날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>항</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>세목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>적요</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>지급계좌</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="hap_recalculator_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>합계 재계산</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_row_Button">
   <property name="geometry">
    <rect>
     <x>789</x>
     <y>73</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modi_row_save_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>29</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1줄 부분 수정</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="accounting_change_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>29</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 계정변경</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>month_widget</tabstop>
  <tabstop>week_widget</tabstop>
  <tabstop>gubun_combo_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>remove_row_Button</sender>
   <signal>clicked()</signal>
   <receiver>cost_modified</receiver>
   <slot>remove_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cost_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>cost_modified</receiver>
   <slot>file_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hap_recalculator_button</sender>
   <signal>clicked()</signal>
   <receiver>cost_modified</receiver>
   <slot>re_calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modi_row_save_button</sender>
   <signal>clicked()</signal>
   <receiver>cost_modified</receiver>
   <slot>row_modi_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accounting_change_button</sender>
   <signal>clicked()</signal>
   <receiver>cost_modified</receiver>
   <slot>accounting_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>data_append()</slot>
  <slot>file_save()</slot>
  <slot>row_modi_save()</slot>
  <slot>re_calculate()</slot>
  <slot>file_close()</slot>
  <slot>remove_row()</slot>
  <slot>cost_serch()</slot>
  <slot>cost_reset()</slot>
  <slot>accounting_change()</slot>
 </slots>
</ui>
